docs/try_argo_cd_locally.md

```diff
@@ -0,0 +1,57 @@
+# Try Argo CD Locally
+
+!!! tip
+    This guide assumes you have a grounding in the tools that Argo CD is based on. Please read [understanding the basics](understand_the_basics.md) to learn about these tools.
+
+
+Follow these steps to install `Kind` for local development and set it up with Argo CD.
+
+To run an Argo CD development environment [review the developer guide for running locally](../developer-guide/running-locally).
+
+## Install Kind
+
+```bash
+curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-darwin-amd64  
```

Comment:
Should we link to installing kind here? We are assuming everyone is running a mac. 

https://kind.sigs.k8s.io/docs/user/quick-start#installation